corehq/apps/hqadmin/reports.py

```diff
@@ -151,23 +152,19 @@ def rows(self):
             return []
 
         rows = []
-        events = navigation_events_by_user(
+        events = all_audit_events_by_user(
             self.selected_user, self.selected_domain, self.datespan.startdate, self.datespan.enddate
         )
         for event in events:
-            rows.append([
-                event.event_date,
-                event.user,
-                event.domain or '',
-                event.ip_address,
-                event.request_method,
-                event.request_path
-            ])
-        return rows
+            row = get_generic_log_event_row(event)
+            rows.append(row)
+
+        # sort by date asc
+        return sorted(rows, key=lambda x: x[0])
 
     @memoized
     def _is_limit_exceeded(self):
-        where = filters_for_navigation_event_query(
+        where = filters_for_audit_event_query(
```

Comment:
Just realized this filter is only applied to `NavigationEventAudit` in this function...